### DISCUSSION TITLE
How to set up a layer that prevents flow in model

### DISCUSSION THREAD

**Discussion by AH-CSM** [modflowpy/flopy#1585] [Q&A] [answered]
I am trying to create a layer that doesn't allow hydraulic flow across it to act as a bottom impermeable base. So far the only thing I can think of based on educated guessing and Google research is to have an extra layer at the bottom of my model with a conductivity so low, it is basically impermeable. Are there not any built-in packages or methods I could use that implement a no-flow boundary on the bottom of my model instead?
Thank you in advance.

**Comment by emorway-usgs** [comment, marked as answer]
Wherever active groundwater model cells are _not_ connected to other active groundwater model cells, then the corresponding cell face is by default a no-flow boundary.  Only when active gw model cells are connected through a shared cell face will groundwater flow occur across a cell face.  

Therefore, the bottom face of the cells in your bottom layer is effectively acting as a no-flow boundary.
